dispatch/static/manager/src/js/pages/EventAudit/EventAuditPagination.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import R from 'ramda'
+
+import LinkButton from '../../components/inputs/LinkButton'
+
+function getPath(location, page) {
+  let query = R.clone(location.query)
+
+  query.page = page
+
+  return {
+    pathname: location.pathname,
+    query: query
+  }
+}
```

Comment:
Is this done anywhere else in the application? Does it make sense to move to `util/helpers.js`?